app/src/main/java/net/squanchy/home/HomeActivity.java

```diff
@@ -51,6 +51,7 @@
 
 import static net.squanchy.google.GoogleClientId.HOME_ACTIVITY;
 
+@SuppressWarnings({"PMD.ExcessiveImports", "PMD.GodClass"})           // Unfortunately this activity has got a lot to do
```

Comment:
I have in my TODO list to track these separately. I'd rather that than increasing the threshold, which might mask new issues if old ones are fixed in the same run.